pkg/worker/request.go

```diff
@@ -34,7 +33,7 @@ import (
 // come.)
 func DoRequest(url string, client *http.Client, callback func() (io.Reader, string, error)) error {
 	input, mimeType, err := callback()
-	pesterClient := pester.NewExtendedClient(client)
+	//pesterClient := pester.NewExtendedClient(client)
```

Comment:
?

Reply:
whoops. Testing stuff I forgot to uncomment!